program_1.rb

```diff
@@ -0,0 +1,31 @@
+<<<<<<< HEAD
```

Comment:
once you received conflict remove. you should remove this code.
otherwise it must be a syntax error